docs/pages/database-access/reference/configuration.mdx

```diff
@@ -111,6 +111,11 @@ spec:
     ...
     -----END CERTIFICATE-----
 
+  # Additional database options.
+  options:
+    # MySQL server version reported by Teleport Proxy in tunnel mode when the real version is not available.
```

Comment:
Is there another way to phrase "the real version"? Why would the MySQL server version not be the real version?

(I haven't looked at this PR beyond this one comment, so apologies if the answer is elsewhere.)

Reply:
I've tried my best 😅